bpf/lib/lb.h

```diff
@@ -1062,6 +1105,12 @@ static __always_inline int lb6_local(const void *map, struct __ctx_buff *ctx,
 
 	ipv6_addr_copy(&tuple->daddr, &backend->address);
 
+	if (lb6_svc_is_l7_punt_proxy(svc)) {
+		if (__lookup_ip6_endpoint(&backend->address)) {
+			ret = LB_PUNT_TO_STACK;
+			goto drop_err;
+		}
+	}
```

Comment:
@borkmann :wave: how is this meant to behave when `lb*local()` is called from `bpf_lxc`?